docs/includes/apiargs-MongoDBCollection-method-dropIndex-option.yaml

```diff
@@ -5,6 +5,8 @@ source:
 source:
```

Comment:
Should maxTimeMS also show `versionadded:: 1.3` here and in dropIndexes-option??

Reply:
Ah, I missed https://github.com/mongodb/mongo-php-library/commit/3e83aa78ef5cf7cd52492e68899f3fe62e98c333. Thanks for catching that!